manage.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python3
+import sys
+import os
+
+
+def print_green(s):
+    GREEN = '\033[92m'
+    ENDC = '\033[0m'
+    print(f"{GREEN}{s}{ENDC}")
+
+
+def run_command(cmd):
+    print_green(cmd)
+    os.system(cmd)
+
+
+def lock(extra_args):
+    """
```

Comment:
https://www.python.org/dev/peps/pep-0257/#one-line-docstrings
[pydocstyle](http://www.pydocstyle.org/en/stable/) can be used to lint these things.

Reply:
It will also suggest that you add docstrings to the other functions.